solidity/test/maintainer/MaintainerProxy.test.ts

```diff
@@ -0,0 +1,3434 @@
+                  )
+                }
+
+                // eslint-disable-next-line @typescript-eslint/no-extra-semi
+                outcome = runRedemptionScenario(data, beforeProofActions)
+              })
+
+              after(async () => {
+                await restoreSnapshot()
+              })
+
+              it("should succeed", async () => {
+                await expect(outcome).to.not.be.reverted
+              })
+
+              it("should refund ETH", async () => {
+                const resolvedOutcome = await outcome
+
+                const postThirdPartyBalance = await provider.getBalance(
+                  authorizedMaintainer.address
+                )
+
+                const diff = postThirdPartyBalance.sub(
+                  resolvedOutcome.initThirdPartyBalance
+                )
+                expect(diff).to.be.gt(0)
+                expect(diff).to.be.lt(
+                  ethers.utils.parseUnits("5500000", "gwei") // 0,0055 ETH
+                )
+              })
+            }
+          )
+        })
+      })
+    })
+  })
+
+  describe("notifyWalletCloseable", () => {
+    before(async () => {
+      await createSnapshot()
+    })
+
+    after(async () => {
+      walletRegistry.requestNewWallet.reset()
+
+      await restoreSnapshot()
+    })
+    context("when called by an unauthorized third party", async () => {
+      it("should revert", async () => {
+        await expect(
+          maintainerProxy
+            .connect(thirdParty)
+            .notifyWalletCloseable(
+              ecdsaWalletTestData.pubKeyHash160,
+              NO_MAIN_UTXO
+            )
+        ).to.be.revertedWith("Caller is not authorized")
+      })
+    })
+
+    context("when called by an authorized maintainer", async () => {
+      context("when the reported wallet is not the active one", () => {
+        context("when wallet is in Live state", () => {
+          before(async () => {
+            await createSnapshot()
+
+            await maintainerProxy
+              .connect(governance)
+              .authorize(walletRegistry.address)
+
+            // Set the deposit dust threshold to 0.0001 BTC, i.e. 100x smaller than
+            // the initial value in the Bridge in order to save test Bitcoins.
+            await bridge.setDepositDustThreshold(10000)
+
+            await bridge.setWallet(ecdsaWalletTestData.pubKeyHash160, {
+              ecdsaWalletID: ecdsaWalletTestData.walletID,
+              mainUtxoHash: ethers.constants.HashZero,
+              pendingRedemptionsValue: 0,
+              createdAt: await lastBlockTime(),
+              movingFundsRequestedAt: 0,
+              closingStartedAt: 0,
+              pendingMovedFundsSweepRequestsCount: 0,
+              state: walletState.Live,
+              movingFundsTargetWalletsCommitmentHash: ethers.constants.HashZero,
+            })
+          })
+
+          after(async () => {
+            walletRegistry.requestNewWallet.reset()
```

Comment:
Why do we need this line?

Reply:
We don't. Removed in https://github.com/keep-network/tbtc-v2/pull/209/commits/000f0dc96c572d263b1035a5dd6cad85f98a4a50.